src/sass/Fabric.scss

```diff
@@ -12,8 +12,10 @@
 @import './Animation';
 @import './BrandIcon';
 @import './Color';
+@import './Font.Definitions';
 @import './Font';
 @import './Grid';
+@import './IconFont';
```

Comment:
@mikewheaton super pedantic preference: would you prefer `Definition` or `Definitions` for the Icon file? There's technically only one definition :/